### DISCUSSION TITLE
Request add new site specific CSS for https://au.acorn.tv

### DISCUSSION THREAD

**Discussion by sstohelit** [W4775/Widify#6] [Ideas]
Request add new site specific CSS for https://au.acorn.tv 
pretty please for ye olde english shows

**Comment by W4775** [comment]
I'm on it! Thanks for the suggestion!

**Reply by W4775** [reply]
I just took a quick look. The trailers look to fill my ultrawide nicely.

Were you just referring to the show listings? Make it something like this?
![image](https://user-images.githubusercontent.com/31900031/221199369-a4554a3b-8cb7-4ff3-9999-fe055a49186b.png)

If not I'll sign up for a trial and check the videos.

**Comment by sstohelit** [comment]
How embarrassing! You're right, already fills the new screen just right.
D'oh!

Thanks heaps for looking. The show listings wide might be cool, whaddya
reckon?

**Reply by W4775** [reply]
Yeah, I could apply the changes I made to get that screenshot above. When I push out the changes it should automatically pick up in the extension. Thanks again for the feedback and for using WIDIFY! I appreciate it!

**Comment by sstohelit** [comment]
Ps thanks for your awesome mahi (work). I just treated myself the AW3423DWF
and now Star Wars looks amazing!

**Reply by W4775** [reply]
Congrats on the new monitor! I personally have a AW3418DW, been itching to upgrade :)
